test-suite/src/test/require-jdk7/org/scalajs/testsuite/javalib/CollectionsTestOnJDK7.scala

```diff
@@ -0,0 +1,62 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.javalib
+
+import java.{util => ju}
+
+import org.scalajs.jasminetest.JasmineTest
+
+object CollectionsTestOnJDK7 extends JasmineTest with ExpectExceptions {
+
+  describe("java.util.Collections.empty* on JDK7 or higher") {
+    it(s"should implement emptyIterator[T](): Iterator[T] ") {
```

Comment:
Spurious space at the end of the title. There are several occurrences in this file.